spec/models/manageiq/providers/cloud_manager/provision_workflow_spec.rb

```diff
@@ -32,6 +43,22 @@
         expect(template_hash[attr]).to eq cloud_init_template.send(attr)
       end
     end
+
+    it "should retrieve sysprep templates when cloning" do
+      options = {'key' => 'value' }
+      allow(sysprep_workflow).to receive(:supports_native_clone?).and_return(true)
+      allow_any_instance_of(MiqTemplate).to receive(:platform).and_return('windows')
```

Comment:
Can this `allow_any_instance_of` be replaced with `allow(template)` to avoid one of the rubocop warnings?

Reply:
Tried that, doesn't work, though I'm not sure why as it definitely looks like it *should* work.

Reply:
That means that the `miq_template` instance is getting loaded through a different path instead of using the in-memory copy created by the `let` block.  I cannot do it today, but I'll find a little time to see if I can find a where we can stub and return the in-memory version to avoid this `any_instance` call.

Reply:
@gmcculloug Ok, pushed up that change, thanks!